src/bilder/images/edxapp/templates/common_values.yml

```diff
@@ -133,7 +133,8 @@ BLOCK_STRUCTURES_SETTINGS:
     TASK_MAX_RETRIES: 5
 BRANCH_IO_KEY: ''
 BUGS_EMAIL: odl-devops@mit.edu  # MODIFIED
-BULK_EMAIL_DEFAULT_FROM_EMAIL: no-reply@{{ key "edxapp/ses-mail-domain" }}  # MODIFIED
+DEFAULT_FROM_EMAIL: mitxonline-support@mit.edu  # MODIFIED
+EMAIL_USE_DEFAULT_FROM_FOR_BULK: true
```

Comment:
Is this setting merged upstream, or is it something that we've been maintaining for our residential/xpro deployments?

Reply:
When we merged `EMAIL_USE_DEFAULT_FROM_FOR_BULK` upstream (https://github.com/edx/edx-platform/pull/24951), we standardized on using `DEFAULT_FROM_EMAIL` instead of  `BULK_EMAIL_DEFAULT_FROM_EMAIL` documented at https://edx.readthedocs.io/projects/edx-platform-technical/en/latest/featuretoggles.html?highlight=bulk#featuretoggle-bulk_email.EMAIL_USE_DEFAULT_FROM_FOR_BULK

However, I haven't been able to locate any documentation on the `DEFAULT_FROM_EMAIL` setting, although it's included in the configuration repo, for example at https://github.com/edx/configuration/blob/b99fb97b161e35829ce02ebec32208bd26ae4336/docker/build/edxapp/lms.yml#L209